packages/lisk-transaction-pool/src/transaction_pool.ts

```diff
@@ -198,14 +213,37 @@ export class TransactionPool {
 		this._queues.verified.enqueueMany(transactions);
 	}
 
+	// It is assumed that signature is verified for this transaction before this function is called
+	public addVerifiedSignature(signatureObject: SignatureObject): boolean {
+		const transaction = this.findInTransactionPool(
+			signatureObject.transactionId,
+		);
+		if (transaction) {
+			return transaction.addSignature(
```

Comment:
we need to decide `addSignature` vs `addMultisignature`

Reply:
I think addMultisignature makes more sense since we add signature for second passphrase on client side as well. Therefore, I'll update this function name.